### ISSUE TITLE
missing color contrast: white text on lightgrey background

### ISSUE THREAD

**Issue by timlg07** [arXiv/html_feedback#4045]
### Description

missing color contrast: white text on lightgrey background

### (Optional:) Please add any files, screenshots, or other information here.

![Image](https://github.com/user-attachments/assets/85a89a1a-58d1-46a0-abd4-ba18f586537b)

https://arxiv.org/html/2504.19287v1

### (Required) What is this issue most closely related to? Select one.

Other

### Internal issue ID

88a7d5cb-8e33-47b0-bdc7-59bc2b2ecf72

### Paper URL

https://arxiv.org/html/2504.19287v1

### Browser

Chrome/133.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
